packages/react-storage/src/components/StorageBrowser/components/DataTable.tsx

```diff
@@ -60,22 +61,24 @@ export function DataTable<
   const { columns, rows } = data ?? {};
 
   return (
-    <TableElement aria-label="Table" className={TABLE_CLASS_NAME}>
-      <TableHeadElement className={`${TABLE_CLASS_NAME}__head`}>
-        <TableRowElement className={TABLE_ROW_CLASS_NAME}>
-          {columns?.map(renderColumnHeaderItem)}
-        </TableRowElement>
-      </TableHeadElement>
-      <TableBodyElement className={`${TABLE_CLASS_NAME}__body`}>
-        {rows?.map((row, index) => (
-          <TableRowElement
-            key={`row-${index}`}
-            className={TABLE_ROW_CLASS_NAME}
-          >
-            {row.map(renderRowDataItem)}
+    <ViewElement className="storage-browser__table-wrapper">
```

Comment:
I'm not sure what the styling issue was here but I don't really agree this base component should always have a wrapping div